sw/airborne/modules/datalink/extra_pprz_dl.c

```diff
@@ -47,13 +47,25 @@
 #include "subsystems/datalink/telemetry.h"
 
 
+
+bool extra_dl_msg_available;
+uint8_t extra_dl_buffer[MSG_SIZE]  __attribute__((aligned));
```

Comment:
Actually, the `dl_buffer` is just a temporary buffer that receives data from transport buffers (one per stream) and calls datalink functions and callback. This is done in sequence so unless the different streams are coming from different threads (and we would have other issues here probably) it is not really needed to have a different buffer. On the other hand, if we don't have memory issues, it shouldn't be a problem to have a second one.